scripts/experimental_helper.sh

```diff
@@ -1,141 +0,0 @@
-#!/bin/bash
```

Comment:
This file is now empty and ideally should be deleted

Reply:
cf #449 so I think we can keep it as it is.